src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs

```diff
@@ -117,17 +119,20 @@ private enum ResponseStatus
                         $" Error Code: {errorCode} \n" +
                         $" Error Message: {authResult.Error.Status} \n" +
                         $" WAM Error Message: {authResult.Error.Context} \n" +
-                        $" Internal Error Code: {internalErrorCode} \n" +
-                        $" Is Retryable: true";
+                        $" Internal Error Code: {internalErrorCode} \n";                        
                     logger.Error($"[WamBroker] WAM_network_error_{errorCode} {errorMessage}");
                     serviceException = new MsalServiceException(errorCode.ToString(), errorMessage);
+                    
                     serviceException.IsRetryable = true;
+                    DecorateExceptionWithRuntimeErrorProperties(serviceException, authResult.Error);
                     throw serviceException;
 
                 default:
                     errorMessage = $"Unknown {authResult.Error} (error code {errorCode}) (internal error code {internalErrorCode})";
```

Comment:
seems like we already have all this info, should we just append the existing error codes and tags appropriately to avoid duplicating and adding a new property bag? 

![image](https://user-images.githubusercontent.com/90415114/235486491-8e5db91a-7991-4fdc-b6a8-c26c22791746.png)

Reply:
We can't expect partners to fetch this data by parsing error messages.